lib/rules/lint-require-valid-alt-text.js

```diff
@@ -80,16 +87,25 @@ module.exports = class A11yLintAltText extends Rule {
               }
 
               if (normalizedAltValue !== null) {
-                const existingWords = REDUNDANT_WORDS.filter(
-                  word => normalizedAltValue.indexOf(word) !== -1
-                );
-                if (existingWords.length) {
+                if (normalizedAltValue.match(/^[0-9]+$/g)){
                   this.log({
-                    message: errorMessage,
+                    message: 'a number is not valid alt text',
```

Comment:
This message doesn't really fit with the other error messages we issue here (they are sentences, but this one isn't).